NuWeather/config.py

```diff
@@ -76,8 +76,14 @@ class NuWeatherGeocode(registry.OnlySomeStrings):
     conf.registerGlobalValue(NuWeather.apikeys, backend,
         registry.String("", _("""Sets the API key for %s.""") % backend, private=True))
 for backend in GEOCODE_BACKENDS:
-    conf.registerGlobalValue(NuWeather.apikeys, backend,
-        registry.String("", _("""Sets the API key for %s.""") % backend, private=True))
+    if backend == 'nominatim':
+        # nominatim doesn't require an API key, but valid_geo_backends checks for existence
+        # of API Keys
+        conf.registerGlobalValue(NuWeather.apikeys, backend,
+            registry.String("1", _("""Sets the API key for %s.""") % backend, private=True))
```

Comment:
Actually, this config option shouldn't exist at all.

Reply:
so how would you suggest going about this one, just let nominatim exist (blank) in apikeys?

or ?
```suggestion
            pass
```

Reply:
```python
# OSM/Nominatim works without API keys
if backend != 'nominatim':
    conf.registerGlobalValue(NuWeather.apikeys, backend,
        registry.String("", _("""Sets the API key for %s.""") % backend, private=True))
```

Reply:
yeah